README.md

```diff
@@ -237,3 +237,78 @@ to us.usd.red.alice
   typeString: 'ilqp_by_destination_response',
  data: { sourceAmount: '10782788022', sourceHoldDuration: 5000 } }
 ```
+
+# <img src="./assets/cat.svg" width="25px"> Cat
+
+BtpCat is strictly for logging. It can be used to display BTP packets, and the ILP packets inside them,
```

Comment:
Because the JSON string representing for instance: https://github.com/interledgerjs/btp-toolbox/blob/2e9822fa3ca08b477287a512af2f9ff61fd787ee/test/catSpec.js#L45-L65 is actually quite ugly, namely:

```js
{ type: 6,
  requestId: 4,
  data:
   [ { protocolName: 'ilp',
       contentType: 0,
       data: <Buffer 06 1d 0f 64 65 2e 65 75 72 2e 62 6c 75 65 2e 62 6f 62 00 00 00 02 18 71 1a 00 00 00 0b b8 00> },
     { protocolName: 'to', contentType: 1, data: 'us.usd.red.connie' } ] }
```

Reply:
Ok that's fair